tests/UIButtons.js

```diff
@@ -0,0 +1,173 @@
+import Menu from "../page-objects/components/Menu";
+import HomePage from "../page-objects/pages/Homepage";
+
+describe("Checking the appearance of buttons", () => {
+  beforeEach(async () => {
+    HomePage.openHomePage();
+    Menu.waitForExistHeader();
+    Menu.clickOnApproveButton();
+    Menu.clickOnApproveButtonFirefox();
+    await browser.pause(5000);
+  });
+
+  it("Checking the appearance of the menu buttons", async () => {
+    const buttons = [
+      Menu.cuteButton,
+      Menu.funnyButton,
+      Menu.goodButton,
+      Menu.omgButton,
+      Menu.moreButton,
+    ];
+
+    for (let i = 0; i < buttons.length; i++) {
+      const elem = buttons[i];
+
+      const fontSize = await elem.getCSSProperty("font-size");
+      console.log(fontSize);
+      const valueFontSize = fontSize.value;
```

Comment:
It is not necessary to create two variables to get the value of the property. You could do something like:
**_const fontSizeValue = await elem.getCSSProperty("font-size").value;_**

This comment can be applied to all the property values from this test.